libs/Requests-2.0.4/src/Transport/Curl.php

```diff
@@ -390,7 +390,7 @@ private function setup_handle($url, $headers, $data, $options) {
 			if ($data_format === 'query') {
 				$url  = self::format_get($url, $data);
 				$data = '';
-			} elseif (!is_string($data)) {
+			} elseif (!is_string($data) && !isset($data["file"])){
```

Comment:
where is the file index because it uses when user defined the "file" name index otherwise it doesn't work